MonoGame.Framework/FrameworkResources.cs

```diff
@@ -0,0 +1,15 @@
+﻿// MonoGame - Copyright (C) The MonoGame Team
+// This file is subject to the terms and conditions defined in
+// file 'LICENSE.txt', which is part of this source code package.
+
+namespace Microsoft.Xna.Framework
+{
+    internal static class FrameworkResources
+    {
+        #region Error strings
+
+        internal const string ResourceCreationWhenDeviceIsNull = "The GraphicsDevice must not be null when creating new resources.";
```

Comment:
I've thought about it before, and is something we should do.  Thanks @Shqrdx for getting it started.